reusable_workflows/repo_policies/bot_checks/check_bot_approved_files.py

```diff
@@ -46,17 +45,11 @@ def get_approved_files(config_file: str) -> list[str]:
     """
     Extracts the list of approved files from the config file.
     """
-    try:
-        config = json.loads(config_file)
-    except json.JSONDecodeError:
-        raise Exception("Config file is not a valid JSON file")
-    try:
-        approved_files = config["approved_files"]
-    except KeyError:
-        raise Exception("No approved_files key found in config file")
-
+    approved_files = [
+        line for line in config_file.splitlines() if not line.strip().startswith("#")
+    ]
     if len(approved_files) == 0:
-        raise Exception("No approved files found in config file")
+        raise ValueError("No approved files found in config file")
```

Comment:
Why do we actually want to error in case there are no approved files? What if we temporarily want to forbid any bot from changing any files by just emptying that file?

Reply:
Ah well I wanted to make the error as informative as possible in case someone made a mistake in setting up the config, but I suppose I could also just let it fail in the next step where it checks if the changed files are within the list of approved files.

Reply:
Yes makes sense. Allowing empty files (or files with only comments) should be allowed I think just like it's allowed for CODEOWNERS.